eng/common/tools.ps1

```diff
@@ -494,6 +494,13 @@ function InitializeBuildTool() {
     }
     $dotnetPath = Join-Path $dotnetRoot (GetExecutableFileName 'dotnet')
     $buildTool = @{ Path = $dotnetPath; Command = 'msbuild'; Tool = 'dotnet'; Framework = 'netcoreapp2.1' }
+
+    # initialize Visual Studio variables if we can
+    try {
+      $msbuildPath = InitializeVisualStudioMSBuild -install:$restore
+    }
+    catch {
+    }
```

Comment:
We're not allowed to modify anything under `eng/common` because that's owned by Arcade.

Reply:
Oh yeah !!! I keep forgetting .  I wonder if we can do this elsewhere, I suppose so.